x-pack/plugins/translations/translations/zh-CN.json

```diff
@@ -2029,6 +2029,8 @@
     "kbnVislibVisTypes.controls.pointSeries.valueAxes.positionLabel": "位置",
     "kbnVislibVisTypes.controls.pointSeries.valueAxes.rotateLabel": "旋转",
     "kbnVislibVisTypes.controls.pointSeries.valueAxes.scaleToDataBoundsLabel": "缩放到数据边界",
+    "kbnVislibVisTypes.controls.pointSeries.valueAxes.scaleToDataBounds.bottomMargin": "底部保证金",
+    "kbnVislibVisTypes.controls.pointSeries.valueAxes.scaleToDataBounds.minNeededBottomMargin": "底部保证金必须大于或等于0",
```

Comment:
Same as comment before, it's not necessary to add the translation here, we will handle that before the release

Reply:
translation removed